charts/multi-casskop/crds/db.orange.com_multicasskops.yaml

```diff
@@ -12,7 +12,6 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
-  version: v2
```

Comment:
why remove the default version ?

Reply:
Default version is the version with `storage: true`